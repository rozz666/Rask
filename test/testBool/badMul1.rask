main() -> void
{
    var x = true * false;
}
