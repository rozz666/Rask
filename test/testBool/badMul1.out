badMul1.rask:3:18: error: function binary*(boolean, boolean) not found